src/shared/scrapers/AUS/SA/index.js

```diff
@@ -46,12 +47,13 @@ const scraper = {
       const $ = await fetch.page(this.url);
       const $table = $('table:first-of-type');
       const $trs = $table.find('tbody > tr:not(:first-child)');
-      const data = { state: scraper.state };
+      const data = { state: this.state };
```

Comment:
This is added automatically to everything returned by the scraper, no need to add it here, can just be an empty obj. Test to be sure